Content.Shared/DoAfter/DoAfterArgs.cs

```diff
@@ -146,6 +147,14 @@ public sealed class DoAfterArgs
     public DuplicateConditions DuplicateCondition = DuplicateConditions.All;
     #endregion
 
+    //TODO: User perf to toggle repeat on specific doafters
+    /// <summary>
+    ///     If true, the doafter will automatically repeat until it's cancelled.
+    ///     Only set this in a do-after event that you would like repeated.
+    /// </summary>
+    [DataField("repeat")]
+    public bool Repeat = false;
```

Comment:
This should just be moved to the do-after itself (i.e., `args.Args.Repeat -> args.Repeat`)